unlock-app/src/hooks/useCertification.ts

```diff
@@ -13,39 +14,62 @@ export const useCertification = ({
   network,
   tokenId,
 }: CertificationProps) => {
+  // Placeholder
+  const placeholderData = {
+    id: '1',
+    network,
+    tokenId: '#', // sets isPlaceholderData
+    owner: `{Recipient's wallet address, or ENS}`,
+    expiration: '{Expiration date}',
+    createdAtBlock: undefined,
+    transactionsHash: ['{Transaction hash}'],
+    lock: {} as any,
+  }
+  const { account } = useAuth()
+
   return useQuery(
```

Comment:
We can display a users' certificate, even if the tokenId is missing because we can easily find it from the connected user's address!